tests/unit/aiplatform/test_metadata.py

```diff
@@ -408,6 +422,51 @@ def query_execution_inputs_and_outputs_mock():
         yield query_execution_inputs_and_outputs_mock
 
 
+_TEST_CLASSIFICATION_METRICS_METADATA = {
+    "confusionMatrix": {
+        "annotationSpecs": [{"displayName": "cat"}, {"displayName": "dog"}],
+        "rows": [{"row": [9, 1]}, {"row": [1, 9]}],
+    },
+    "confidenceMetrics": [
+        {"confidenceThreshold": 0.9, "recall": 0.1, "falsePositiveRate": 0.1},
+        {"confidenceThreshold": 0.5, "recall": 0.5, "falsePositiveRate": 0.7},
+        {"confidenceThreshold": 0.1, "recall": 0.9, "falsePositiveRate": 0.9},
+    ],
+}
+
+_TEST_CLASSIFICATION_METRICS_ARTIFACT_RESOURCE = GapicArtifact(
+    name=_TEST_ARTIFACT_NAME,
+    display_name=_TEST_CLASSIFICATION_METRICS["display_name"],
+    schema_title=constants.GOOGLE_CLASSIFICATION_METRICS,
+    schema_version=constants._DEFAULT_SCHEMA_VERSION,
+    metadata=_TEST_CLASSIFICATION_METRICS_METADATA,
+)
+
+_TEST_CLASSIFICATION_METRICS_ARTIFACT = artifact.Artifact._empty_constructor()
+_TEST_CLASSIFICATION_METRICS_ARTIFACT._gca_resource = (
+    _TEST_CLASSIFICATION_METRICS_ARTIFACT_RESOURCE
+)
+
+
+@pytest.fixture
+def create_classification_metrics_artifact_mock():
```

Comment:
Please update to mock the underlying GAPIC clients and not the Vertex SDK objects. We want to ensure we are capturing the dependency between the Vertex SDK objects in the unit tests.

Reply:
Done